dataland-frontend/tests/e2e/specs/lksg/DataIntegrity.ts

```diff
@@ -37,8 +37,11 @@ describeIf(
      * validates that the data uploaded via the function `uploadLksgDataViaForm` is displayed correctly for a company
      * @param companyId the company associated to the data uploaded via form
      */
-    function validateFormUploadedData(companyId: string): void {
-      cy.visit("/companies/" + companyId + "/frameworks/" + DataTypeEnum.Lksg);
+    function validateFormUploadedData(companyId: StoredCompany): void {
```

Comment:
Input should be "storedCompany", and not "companyId"

Reply:
Yeah of course. Changed